### ISSUE TITLE
Keep calling service after stop or restart process

### ISSUE THREAD

**Issue by chameleonbr** [ice-services/moleculer#48]
Hi, I like this project concept. I've been created 2 projects a worker and gateway connecting on the Redis transport, but I have some problem when I stop or restart worker. The gateway continues calling the stopped worker, look the gateway log messages:

[BROKER] 'api-gw' service is registered!
[API-GW-SVC] API Gateway created!
[API-GW-SVC] API Gateway listening on http://0.0.0.0:3000
[TX] Redis-sub connected!
[TX] Redis-pub connected!
[BROKER] Broker started. NodeID: GW-HPDEV-24176
[TRANSIT] Node 'WORKER-HPDEV-24155' connected! **-> I start the worker**
[TRANSIT] Node 'WORKER-HPDEV-24192' connected! **-> Restart the worker**
[TRANSIT] Node 'WORKER-HPDEV-24155' disconnected!  **-> detected the disconnection**
[API-GW-SVC]   Call 'math.teste' action with params: {}
[BROKER] Action 'math.teste' call timed out on 'WORKER-HPDEV-24155'! **-> look, he called the killed process**
[BROKER] Recall 'math.teste' action (retry: 3)...
[API-GW-SVC]   Call 'math.teste' action with params: {}

After some retries he call right service.
This is a bug or wrong configuration?
```
let broker = new ServiceBroker({
    nodeID: 'GW-' + hostname + '-' + process.pid,
    logger: console,
    metrics: true,
    requestTimeout: 5000,
    requestRetry: 3,
    circuitBreaker: {
        enabled: true,
        maxFailures: 3,
        halfOpenTime: 500,
        failureOnTimeout: true,
        failureOnReject: true
    },
    transporter: new RedisTransporter({
        redis: {
            port: 6379,
            host: 'localhost',
            family: 4,
            db: 1
        }
    })
})

// Load API Gateway
broker.createService(ApiService);

// Start server
broker.start()
```

**Comment by icebob** [comment]
Thanks, I will try it!

**Comment by chameleonbr** [comment]
v0.7.6

**Comment by icebob** [comment]
Thx. I tried and yes it is a bug in v0.7.6. But it is fixed in `master` so in the next version it will work.

**Comment by icebob** [comment]
Fixed and released in v0.7.8
